### DISCUSSION TITLE
Main or dev before cloning

### DISCUSSION THREAD

**Discussion by alecgargett** [simona-cancian/Tafe24S1T101_Vir#4] [General]
Hi Simona, did you clone main or development before you started work on the Mortage Calc? Is your feature branch from main or from development? I can't see where this is specified in the instructions. They only say to "clone the repository" and don't say which branch to clone.

**Comment by simona-cancian** [comment]
Hi Alec, I did it this way:
- cloned the project and checkout **development**
- created the feature_mortgage_calculator from development. 
- after development pushed it to the repository. 
- merged it to development
- after merging pushed it to the repository

**Comment by alecgargett** [comment]
Thanks
